zhaquirks/philips/__init__.py

```diff
@@ -145,8 +145,7 @@ class PhilipsRemoteCluster(CustomCluster):
                 "param2": t.uint24_t,
                 "press_type": t.uint8_t,
                 "param4": t.uint8_t,
-                "param5": t.uint8_t,
-                "param6": t.uint8_t,
+                "duration": t.uint16_t,
```

Comment:
This seems to be supported by many(/all?) of Philips' button devices. It's measured in 10th of seconds.

Confirmed that for ROM001, RWL022, RDM002.

Reply:
Also same for RDM001

Reply:
Is there a chance that you also have a `RWL020` or `RWL021` dimmer (one of the older ones) to test this with?

If not, I'll search for one of mine to test this with.

Reply:
RWL020 is the first gen Dimmer switch ?
That i have in the box for bad Zigbee devices.
@fholzer If you is near W1110 you can having my for testing its easy picking up then im at home if you like (U3 Enkplatz).

MW

Reply:
Yeah, that would be appreciated. From the RWLs I only have the RWL022. My email is ferdinand.holzer ( at ) gmail.com. Get in touch and we can arrange a time later today if possible.

Reply:
Mailed !!